browse/domain/clickthrough.py

```diff
@@ -0,0 +1,28 @@
+"""Functions to create hashes for Click through links.
```

Comment:
I think this should be moved out of domain and into a util directory.

Reply:
I've moved this.